src/main/java/uk/gov/mca/beacons/api/services/DeleteBeaconService.java

```diff
@@ -25,22 +28,33 @@
   private final BeaconGateway beaconGateway;
   private final UserGateway userGateway;
   private final NoteGateway noteGateway;
+  private final OwnerGateway ownerGateway;
+  private final EmergencyContactGateway emergencyContactGateway;
+  private final UseGateway useGateway;
 
   @Autowired
   public DeleteBeaconService(
     BeaconGateway beaconGateway,
     UserGateway userGateway,
-    NoteGateway noteGateway
+    NoteGateway noteGateway,
+    OwnerGateway ownerGateway,
+    EmergencyContactGateway emergencyContactGateway,
+    UseGateway useGateway
   ) {
     this.beaconGateway = beaconGateway;
     this.userGateway = userGateway;
     this.noteGateway = noteGateway;
+    this.ownerGateway = ownerGateway;
+    this.emergencyContactGateway = emergencyContactGateway;
+    this.useGateway = useGateway;
   }
 
   public void delete(DeleteBeaconRequestDTO request) {
     final User user = userGateway.getUserById(request.getUserId());
     if (user == null) throw new UserNotFoundException();
 
+    final var beaconIdToBeDeleted = request.getBeaconId();
+
     final Note note = Note
       .builder()
       .beaconId(request.getBeaconId())
```

Comment:
You can replace this call here with the variable `beaconIdToBeDeleted`

Reply:
changes made